data/locale/fr-FR.ini

```diff
@@ -4,22 +4,22 @@ NDIPlugin.SourceProps.SourceName="Source"
 NDIPlugin.SourceProps.Bandwidth="Bandwidth"
 NDIPlugin.SourceProps.Sync="Synchronisation"
 NDIPlugin.SourceProps.HWAccel="Autoriser l'accélération matérielle"
-NDIPlugin.SourceProps.AlphaBlendingFix="Correction du compositing alpha (ajoute un filtre à la source)"
+NDIPlugin.SourceProps.AlphaBlendingFix="Correction de simulation de transparence (ajoute un filtre à la source)"
 NDIPlugin.SourceProps.ColorRange="Gamme de couleurs YUV"
 NDIPlugin.SourceProps.ColorRange.Partial="Partielle"
 NDIPlugin.SourceProps.ColorRange.Full="Complète"
-NDIPlugin.SourceProps.ColorSpace="Espace de couleurs YUV"
+NDIPlugin.SourceProps.ColorSpace="Espace colorimétrique YUV"
 NDIPlugin.SourceProps.Latency="Latence"
-NDIPlugin.SourceProps.Latency.Normal="Normale (sûr)"
+NDIPlugin.SourceProps.Latency.Normal="Normale (fiable)"
 NDIPlugin.SourceProps.Latency.Low="Réduite (expérimental)"
 NDIPlugin.BWMode.Highest="Haut débit"
 NDIPlugin.BWMode.Lowest="Bas débit"
 NDIPlugin.BWMode.AudioOnly="Audio seulement"
 NDIPlugin.SyncMode.NDITimestamp="Réseau"
-NDIPlugin.SyncMode.NDISourceTimecode="Timing de la source"
+NDIPlugin.SyncMode.NDISourceTimecode="Code temporel de la source"
 NDIPlugin.OutputName="Sortie NDI™"
 NDIPlugin.OutputProps.NDIName="Nom de la sortie"
-NDIPlugin.FilterProps.NDIName="Nom NDI"
+NDIPlugin.FilterProps.NDIName="Nom NDI™"
```

Comment:
👍 
But there appear to be a number of others missing in all other languages too.